Gemfile

```diff
@@ -7,7 +7,7 @@ end
 
 
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
-gem 'rails', '~> 5.2.1'
+gem 'rails', '>= 5.2.2.1'
```

Comment:
`>=` (greater than or equal to) acts differently from `~>` (pessimistic constraint).

`>= 5.2.2.1` will allow `5.2.2.1` but also `5.3` and `6.0` `99999.99.9.9`. This could break us if we're not ready to upgrade to Rails 6 or Rails 99999.

You may want to have `~> 5.2.2.1`, which will allow `5.2.2.1` and `5.2.2.9` but not `5.2.3` or above.

Reply:
https://stackoverflow.com/questions/5101591/what-does-mean-in-a-gem-file

Reply:
Thanks for the insight :)